tests/cli/test_cli.py

```diff
@@ -816,11 +816,11 @@ def test_upload3(ctx):
 
     # Upload URL that's an archive
     uuid = _run_command([cl, 'upload', 'http://alpha.gnu.org/gnu/bc/bc-1.06.95.tar.bz2'])
-    check_contains(['README', 'INSTALL', 'FAQ'], _run_command([cl, 'cat', uuid]))
+    check_contains(['bc-1.06.95'], _run_command([cl, 'cat', uuid]))
```

Comment:
Why this change?

Reply:
See the description of the case with http://alpha.gnu.org/gnu/bc/bc-1.06.95.tar.bz2 in this PR description. It contains a single directory, bc-1.06.95.